Package.swift

```diff
@@ -1,36 +1,63 @@
 // swift-tools-version:5.4
 import PackageDescription
 
-var packageDependencies: [Package.Dependency] = []
-var calDependencies: [Target.Dependency] = ["AwsCCommon"]
-var ioDependencies: [Target.Dependency] = ["AwsCCommon", "AwsCCal"]
-
-#if os(Linux)
-packageDependencies.append(.package(name: "S2N", path: "./S2N"))
-
-ioDependencies.append(.product(name: "S2N", package: "S2N"))
-calDependencies.append(.product(name: "LibCrypto", package: "S2N"))
-#endif
+let excludesFromAll = ["tests", "cmake", "CONTRIBUTING.md",
+                       "LICENSE", "format-check.sh", "NOTICE", "builder.json",
+                        "CMakeLists.txt", "README.md"]
+var packageTargets: [Target] = []
 
 var package = Package(name: "AwsCrt",
     platforms: [.iOS(.v11), .macOS(.v10_14)],
     products: [
       .library(name: "AwsCommonRuntimeKit", targets: ["AwsCommonRuntimeKit"]),
       .executable(name: "Elasticurl", targets: ["Elasticurl"])
-    ],
-    dependencies: packageDependencies
+    ]
 )
 
-let excludesFromAll = ["bin", "tests", "cmake", "codebuild", "CONTRIBUTING.md",
-                       "LICENSE", "format-check.sh", "NOTICE", "builder.json",
-                       "sanitizer-blacklist.txt", "CMakeLists.txt", "README.md",
-                       "CODE_OF_CONDUCT.md", "build-deps.sh"]
+var calDependencies: [Target.Dependency] = ["AwsCCommon"]
+var ioDependencies: [Target.Dependency] = ["AwsCCommon", "AwsCCal"]
+
+#if os(Linux)
```

Comment:
is freebsd or netbsd supported for spm yet?

Reply:
not that I am aware of, os options here: https://docs.swift.org/swift-book/ReferenceManual/Statements.html#:~:text=Valid%20arguments-,os(),-macOS%2C%20iOS%2C%20watchOS